packages/vm/src/testing/contract.rs

```diff
@@ -32,13 +37,19 @@ impl Contract {
         let engine = make_compiling_engine(memory_limit);
         let module = compile(&engine, wasm)?;
         let storage = MockStorage::new();
-        let contract = Self { module, storage };
+        let contract = Self {
+            engine,
+            module,
+            storage,
+        };
         Ok(contract)
     }
 
     /// change the wasm code for testing migrate
     ///
     /// call this before `generate_instance` for testing `call_migrate`.
+    ///
+    /// the engine and module need to correspond one-to-one, and with changes in WASM, both the engine and module are being updated.  
```

Comment:
keep 80 or less letters in one line and delete tailing spaces.

Reply:
Fixed, thank you.